### ISSUE TITLE
Value topName in class TreadleTester cannot be accessed in treadle.TreadleTester

### ISSUE THREAD

**Issue by hadirkhan10** [ucb-bar/chisel-gui#26]
Running sbt assembly in chisel-gui folder results in the following error on Ubuntu VM running on Windows 8:
![Screenshot from 2020-02-13 20-14-17](https://user-images.githubusercontent.com/41375553/74500986-8ff12900-4e9d-11ea-93e5-7228f599321b.png)

**Comment by chick** [comment]
Yes, the topName field has been added to the treadle api, but that update to treadle 1.4 has not been published yet. It will be soon in the meantime you would need to git clone treadle and do a publishLocal on the master branch. This should be fixed soon

**Comment by hadirkhan10** [comment]
I am not sure what does "git clone treadle" mean. Should I clone the Treadle repository separately and then attach it here somehow?

**Comment by chick** [comment]
I haven't figured this out yet. In general yes, one would typically do the following in the powershell.
This assumes you have some directory `workspace` where you are building things 
```
cd workspace
git clone https://github.com/freechipsproject/treadle
cd treadle
sbt publishLocal
cd ../chisel-gui
sbt
runMain visualizer.ChiselGUI samples/DecoupledGcd.fir --vcd samples/DecoupledGcd.vcd
```
But I tried this and did not seem to pick up the local published jar for treadle. I'm not that familiar with windows, so I'm still trying to figure what's going on

**Comment by hadirkhan10** [comment]
So for the time being what should I do to make it work. If you can provide specific commands to resolve this error, it would be a great help.
